include/Quaternion.hpp

```diff
@@ -34,12 +42,6 @@ struct Quaternion
     Quaternion                  slerp       (const Quaternion& target, const f32 t) const noexcept;
     Quaternion                  nlerp       (const Quaternion& target, const f32 t) const noexcept;
```

Comment:
constexpr missing

Reply:
`Quaternion::slerp` and `Quaternion::nlerp` cannot be `constexpr` because they make use of some of `<cmath>` functions which are not `constexpr`, or of another non-`constexpr` method